### DISCUSSION TITLE
is it possible to fix identified persons from multiple photos?

### DISCUSSION THREAD

**Discussion by santiagozky** [photoprism/photoprism#3839] [Q&A] [answered]
I am just trying photoprism. So far I like it but the face detection is not particularly reliable. I added around 2500 photos to start with and 2 children in my library got seriously mixed up. so I have a two questions on the topic

- Does photoprism does any kind of continuous scanning? i.e. will it fix incorrected identified people later on, if I have more photos?
- Can I ask photoprism to "rescan" faces on an album or set of photos once I have more photos (and hopefully more reliable data)?
- Is there a way to set the identified person when I select multiple photos ? (I know it can be done for a single photo). I understnad this would be possible only in photos with a single person.

**Comment by graciousgrey** [comment, marked as answer]
Thank you very much for your feedback :)

> Does photoprism does any kind of continuous scanning? i.e. will it fix incorrected identified people later on, if I have more photos?

PhotoPrism does constantly update the face clusters in the background. If you add additional photos or manually assign names to faces or correct wrong namings, PhotoPrism uses this information automatically to improve the recognition.

>Can I ask photoprism to "rescan" faces on an album or set of photos once I have more photos (and hopefully more reliable data)?

That's not needed as it happens automatically in the background. 

>Is there a way to set the identified person when I select multiple photos ? (I know it can be done for a single photo). I understnad this would be possible only in photos with a single person.

That's not possible yet but many improvements are planned:

- https://github.com/photoprism/photoprism/issues/1679
- https://github.com/photoprism/photoprism/issues/1601

**Reply by santiagozky** [reply]
thank you. I'll try to add more photos and correct the most obvious mistakes to see how it improves.
One last question.
Does photoprism handles different people I've tagged manually vs people tagged automatically? (e.g. maybe manually tagged people is never corrected since

**Reply by graciousgrey** [reply]
Faces you have tagged manually should not be overwritten.
